api/src/main/java/org/openmrs/module/patientqueueing/mapper/PatientQueueMapper.java

```diff
@@ -35,6 +35,8 @@
 	private String status;
 	
 	private String dateCreated;
+
+	private String dateChanged;
```

Comment:
Don't you want to also know who changed?

Reply:
Not really. But let me add it.

Reply:
Done responding to this comment